app/components/calls_to_action/adviser_block_component.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= tag.div(class: "image-block") do %>
+  <%= image_pack_tag("images/#{@image}", alt: "") %>
+
+  <%= tag.h2(class: "heading--overlap heading--highlight-#{@title_color} heading--margin-top-0 heading-xl") do %>
+    <%= tag.span(@title) %>
+  <% end %>
+
+  <div class="promos">
+    <%= @text %>
+    <p></p>
```

Comment:
don't use an empty paragraph for spacing - use styles (margins + padding) instead